<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FenViewer</class>
 <widget class="QWidget" name="FenViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>612</width>
    <height>740</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>FEN Viewer</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="f_board">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>600</width>
       <height>600</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>600</width>
       <height>600</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="le_fen_string">
     <property name="placeholderText">
      <string>FEN String</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="le_move_string">
     <property name="placeholderText">
      <string>Move String</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QCheckBox" name="cb_show_all_moves">
        <property name="text">
         <string>show all moves</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="l_next_move_string">
        <property name="text">
         <string>Next Move: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="l_next_move">
        <property name="text">
         <string>None</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb_next_move">
        <property name="text">
         <string>Next Move</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
